React/Views/RCTAnimationType.h

```diff
@@ -16,4 +16,6 @@ typedef NS_ENUM(NSInteger, RCTAnimationType) {
   RCTAnimationTypeEaseOut,
   RCTAnimationTypeEaseInEaseOut,
   RCTAnimationTypeKeyboard,
```

Comment:
Remove this - there are no call sites for it on the native side except for the one's you've updated in this diff.